kafka-rest/src/test/java/io/confluent/kafkarest/controllers/ReplicaManagerImplTest.java

```diff
@@ -43,73 +56,109 @@
 
   private static final String CLUSTER_ID = "cluster-1";
   private static final String TOPIC_NAME = "topic-1";
-  private static final int PARTITION_ID = 0;
+  private static final int PARTITION_ID_1 = 0;
+  private static final int PARTITION_ID_2 = 1;
+  private static final int BROKER_ID_1 = 1;
 
-  private static final PartitionReplica REPLICA_1 =
+  private static final Broker BROKER_1 =
+      new Broker(CLUSTER_ID, BROKER_ID_1, "1.2.3.4", 5, /* rack= */ null);
+
+  private static final PartitionReplica REPLICA_1_1 =
       new PartitionReplica(
           CLUSTER_ID,
           TOPIC_NAME,
-          PARTITION_ID,
-          /* brokerId= */ 1,
+          PARTITION_ID_1,
+          /* brokerId= */ BROKER_ID_1,
           /* isLeader= */ true,
           /* isInSync= */ false);
-  private static final PartitionReplica REPLICA_2 =
+  private static final PartitionReplica REPLICA_1_2 =
       new PartitionReplica(
           CLUSTER_ID,
           TOPIC_NAME,
-          PARTITION_ID,
+          PARTITION_ID_1,
           /* brokerId= */ 2,
```

Comment:
nit - do we have brokerId `2` and `3` as final ints too ?

Reply:
Done.